### PR TITLE
refactor(minor-service-registry)!: refactor weights

### PR DESCRIPTION
**Pull request by cjcobb23**
* handle all bonding, weighting and authorization logic in the service registry

https://axelarnetwork.atlassian.net/browse/AXE-3277

### CODE REVIEW

File: contracts/service-registry/src/state.rs [hunk 1]
```diff
@@ -38,30 +38,38 @@ pub struct Worker {
 }
 
 #[cw_serde]
-pub struct WeightedWorker {
```
Comment (cjcobb23) [inline contracts/service-registry/src/state.rs:41]: switch back to this name

File: contracts/service-registry/src/state.rs [hunk 2]
```diff
@@ -38,30 +38,38 @@ pub struct Worker {
...
+            BondingState::Bonded { amount }
+                if amount >= min_bond
+                    && worker.authorization_state == AuthorizationState::Authorized =>
+            {
+                Some(Self {
+                    worker_info: worker,
+                    weight: WORKER_WEIGHT,
+                })
+            }
+            _ => None,
```
Comment (cjcobb23) [inline contracts/service-registry/src/state.rs:60]: move back to query

File: contracts/voting-verifier/src/execute.rs [hunk 3]
```diff
@@ -1,6 +1,5 @@
 use cosmwasm_std::{
-    to_json_binary, Deps, DepsMut, Env, MessageInfo, QueryRequest, Response, Storage, WasmMsg,
-    WasmQuery,
+    to_binary, Deps, DepsMut, Env, MessageInfo, QueryRequest, Response, Storage, WasmMsg, WasmQuery,
```
Comment (fish-sammy) [inline contracts/voting-verifier/src/execute.rs:2]: Why change it to use the deprecated instead?

Comment (cjcobb23) [reply contracts/voting-verifier/src/execute.rs:2]: unintentional

**Review by cgorenflo — APPROVED** [review]

**Pull request merged**